src/core/runtime.d

```diff
@@ -300,6 +301,30 @@ struct Runtime
      * value of this routine indicates to the runtime whether the tests ran
      * without error.
      *
+     * There are two options for handlers. The `bool` version is deprecated but
+     * will be kept for legacy support. Returning `true` from the handler is
+     * equivalent to returning `0` from the `int` version. Returning `false`
+     * from the handler is equivalent to returning `int.min` from the `int`
+     * version.
+     *
+     * If the routine returns 0, this means nothing is printed, and the main
+     * function is executed. This is generally used to signify that no unit
+     * tests were run.
+     *
+     * If `int.min` or `int.max` is returned, it means the runtime should not
+     * execute main, but return failure or success to the OS respectively.
+     *
+     * If a positive number is returned, that number is used to
+     * print a message giving the value as the number of modules that passed
+     * unit tests. `main` is not executed, and success is returned.
+     *
+     * If a negative number is returned, that number is printed as the number
+     * of modules that failed unit tests. `main` is not executed, and failure
+     * is returned.
+     *
+     * The default unit tester does not return `int.min` or `int.max`, but
+     * rather the number of tests passed or failed (or 0 if none are run).
+     *
```

Comment:
I will still make the return be int, but also define an enum that can be returned for convenience.